Game.cpp

```diff
@@ -0,0 +1,95 @@
+// C++ program to implement tic tac toe game 
+#include <iostream> 
+using namespace std; 
+  
+// Function to draw the Tic-Tac-Toe board 
+void drawBoard(char board[3][3]) 
+{ 
+    cout << "-------------\n"; 
+    for (int i = 0; i < 3; i++) { 
+        cout << "| "; 
+        for (int j = 0; j < 3; j++) { 
+            cout << board[i][j] << " | "; 
+        } 
+        cout << "\n-------------\n"; 
+    } 
+} 
+  
+// Function to check for a win 
+bool checkWin(char board[3][3], char player) 
+{ 
+    // Check rows, columns, and diagonals 
+    for (int i = 0; i < 3; i++) { 
+        if (board[i][0] == player && board[i][1] == player 
+            && board[i][2] == player) 
+            return true; 
+        if (board[0][i] == player && board[1][i] == player 
+            && board[2][i] == player) 
+            return true; 
+    } 
+    if (board[0][0] == player && board[1][1] == player 
+        && board[2][2] == player) 
+        return true; 
+    if (board[0][2] == player && board[1][1] == player 
+        && board[2][0] == player) 
+        return true; 
+    return false; 
+} 
+  
+int main() 
+{ 
```

Comment:
When checking for a win, consider combining similar conditions (rows, columns, diagonals) to make the code more concise and readable.